runtime/runtime-params-estimator/src/lib.rs

```diff
@@ -399,28 +399,53 @@ fn action_add_function_access_key_base(ctx: &mut EstimatorContext) -> GasCost {
 }
 
 fn action_add_function_access_key_per_byte(ctx: &mut EstimatorContext) -> GasCost {
-    let total_cost = {
-        let many_methods: Vec<_> = (0..1000).map(|i| format!("a123456{:03}", i)).collect();
+    let base_cost = action_add_function_access_key_base(ctx) + action_sir_receipt_creation(ctx);
+
+    // Set up estimation with varying method length and total bytes.
+    let mut estimate = |method_len: usize, total_len: usize| {
+        // Nothing prevents a key to list the same method many times. Performance should not be affected.
+        let method_name = "x".repeat(method_len);
```

Comment:
I'd maybe still made names different, just in case something does change in the future impl.

Reply:
Well, if we make names different, names need to be longer than one character.
But an attacker is not restricted to that today, so it is totally feasible to submit 1000x "x". And we estimate that case as slightly more gas heavy than fewer methods with longer names.